.github/workflows/ci.yml

```diff
@@ -208,6 +208,35 @@ jobs:
           UBSAN_OPTIONS: halt_on_error=1
         run: |
           time make test262
+  linux-tcc:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        buildType: [Debug, Release]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: install TCC
+        run: |
+          pushd /tmp
+          git clone --depth 1 https://repo.or.cz/tinycc.git
+          cd tinycc
+          git checkout 9d2068c6309dc50dfdbbc30a5d6757683d3f884c
+          ./configure
+          make
+          sudo make install
+          tcc -v
+          popd
+      - name: build
+        env:
+          CC: tcc
+        run: |
+          make BUILD_TYPE=${{matrix.buildType}}
```

Comment:
You probably don't need a debug/release matrix here. tcc's `-O` option is almost literally a no-op, the only thing it does is define `__OPTIMIZE__`.